skpro/classification/sklearn_classifiers/_sklearn_proba_class.py

```diff
@@ -0,0 +1,144 @@
+"""Adapter to sklearn multiclass classification using Histograms."""
+
+__author__ = ["ShreeshaM07"]
+
+import numpy as np
+import pandas as pd
+
+from skpro.distributions import Histogram
+from skpro.regression.base import BaseProbaRegressor
+from skpro.utils.sklearn import prep_skl_df
+
+
+class SklearnProbaClassifier(BaseProbaRegressor):
+    """A multiclass classifier fitting a histogram distribution."""
+
+    _tags = {
+        "authors": ["ShreeshaM07"],
+        "maintainers": ["ShreeshaM07"],
+        "capability:multioutput": False,
+        "capability:missing": True,
+    }
+
+    def __init__(self, clf, bins=10):
+        self.clf = clf
+        if isinstance(bins, int) or isinstance(bins, np.integer):
```

Comment:
never change `self.x` to be sth different than arg `x`. If you want to do this, use `self._bins`.

Reply:
Yes I did not notice that thanks.